joomla/joomla-cms
# [5.0] Move from static Text access to the text function of the plugin

### Summary of Changes
The `$this->text()` function should be used in plugin template files to translate strings. This eliminates the static use of `Text::_`. When template files start to use functionality from the current context it will allow them to be reusable.

### Testing Instructions
- Create an url custom field
- Open TinyMCE by editing an article
- Check if the strings in the editor buttons are translated

### Actual result BEFORE applying this Pull Request
All strings are translated.

### Expected result AFTER applying this Pull Request
All strings are translated.

### Link to documentations
Please select:
